test/common/concepts_common.h

```diff
@@ -235,20 +235,51 @@ template <typename T, typename F = T> using WithFeederOperatorRoundBracketsNonCo
 template <typename T, typename F = T> using WithFeederWrongFirstInputOperatorRoundBrackets = ParallelForEachFeederBody<T, F, /*() = */State::incorrect_first_input>;
 template <typename T, typename F = T> using WithFeederWrongSecondInputOperatorRoundBrackets = ParallelForEachFeederBody<T, F, /*() = */State::incorrect_second_input>;
 } // namespace parallel_for_each_body
-namespace container_based_sequence {
+namespace parallel_sort_value {
+template<bool MovableV, bool MoveAssignableV, bool ComparableV>
+struct ParallelSortValue
+{
+    ParallelSortValue(ParallelSortValue&&) requires MovableV {}
+    ParallelSortValue& operator=(ParallelSortValue&&) requires MoveAssignableV {return *this;}
+
+    friend bool operator<(const ParallelSortValue&, const ParallelSortValue&) requires ComparableV { return true; }
+};
+
+template<bool MovableV, bool MoveAssignableV, bool ComparableV>
+void swap(ParallelSortValue<MovableV, MoveAssignableV, ComparableV>&,
```

Comment:
IMHO, this function can be removed, because it does not takes any profit into the testing - we do not test if swap is not defined. Also, it is hard to imagine how we can define the class which is move constructible and assignable but not swappable (due to standard implementation of `std::swap` which is based on move ctor and assignment operator). So I would suggest to remove it.